tg_bot.py

```diff
@@ -250,16 +259,36 @@ def print_unsubscribe_button(msg):
         reply_markup=InlineKeyboardMarkup(custom_keyboard, one_time_keyboard=True))
 
 
+def print_subscription_status(update, msg, context): 
```

Comment:
See `start_interval_checking` example, `msg` is not needed in parameters, because if can be retrieved from `update`: `msg = update.message`

Reply:
Removed msg from parameters and retrieved it in the beginning of method.